### PR TITLE
docs: add sample to use view with compose

### PR DESCRIPTION
**Pull request by MahmoudMabrok**

### CODE REVIEW

**Comment by PierfrancescoSoffritti** [comment]
Thanks for moving the PR to the dev branch.

The example should not be at the end of the readme file. Can you put it where the other examples are?

**Comment by MahmoudMabrok** [comment]
Could you consult a place as I was not sure where to add it?

File: README.md [hunk 1]
```diff
@@ -152,6 +153,35 @@ In order to start using the player you need to add a [YouTubePlayerView](#youtub
 </LinearLayout>
 ```
 
+## Use with Jetpack Compose
```
[the remark this answers is not in the document]
Comment (PierfrancescoSoffritti) [reply README.md:156]: And rename section to "Jetpack Compose"

[the remark this answers is not in the document]
Comment (PierfrancescoSoffritti) [reply README.md:156]: Also add entry for XML and Jetpack Compose in the table of content

File: README.md [hunk 2]
```diff
@@ -152,6 +153,35 @@ In order to start using the player you need to add a [YouTubePlayerView](#youtub
...
+            }
+        }
+
+        AndroidView(
+            {
+                youtubePlayer
+            }, modifier = Modifier
+                .fillMaxWidth()
+                .wrapContentHeight()
+        )
```
Comment (shubhamsinghshubham777) [inline README.md:181]: Would it make sense to wrap the AndroidView inside a DisposableEffect? Something like this:
```
DisposableEffect(
    AndroidView(
        modifier = Modifier.fillMaxWidth(), 
        factory = { youtubePlayerView }
    ), 
    effect = { onDispose { youtubePlayerView.release() } }
)
```

**Comment by PierfrancescoSoffritti** [comment]
Closing as the comments have never been addressed. I have opened #935 to keep track of this.

**Pull request closed without merging**